### PR TITLE
Update git instructions for install on Debian/Ubuntu based machines

### PR DESCRIPTION
**Pull request by shakaran**
I note that the python2 dependency seems wrong in ubuntu based machines (it should be python2.7 at least), and same for autotools, the package is named autotools-dev. Probably in Debian are the old names, but I cannot ensure, only I can ensure for Ubuntu 16.10.

Also adding the -y to apt-get commands, so it is more easy to just copy & paste.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,9 +28,13 @@ License: ISC, see [COPYING](./COPYING) for details.
...
-    $ sudo apt install build-essential
-    $ sudo apt install libjson-c-dev libgirepository1.0-dev libglib2.0-dev
-    $ sudo apt install python2 autotools intltool gettext   # Building from git
+    $ sudo apt install -y build-essential
+    $ sudo apt install -y libjson-c-dev libgirepository1.0-dev libglib2.0-dev
+    $ sudo apt install -y python2 autotools intltool gettext   # Building from git
+
+The last line for Ubuntu systems should be:
+
+    $ sudo apt install -y python2.7 autotools-dev intltool gettext
```
Comment (achadwick) [inline README.md]: Good catch, but it's `python2.7` recent Debian too. Please can you rephrase?

Comment (shakaran) [inline README.md]: Done

File: README.md [hunk 2]
```diff
@@ -28,9 +28,13 @@ License: ISC, see [COPYING](./COPYING) for details.
 On recent Debian-like systems, you can type the following
 to get started with a standard configuration:
 
-    $ sudo apt install build-essential
-    $ sudo apt install libjson-c-dev libgirepository1.0-dev libglib2.0-dev
-    $ sudo apt install python2 autotools intltool gettext   # Building from git
+    $ sudo apt install -y build-essential
+    $ sudo apt install -y libjson-c-dev libgirepository1.0-dev libglib2.0-dev
+    $ sudo apt install -y python2.7 autotools intltool gettext   # Building from git
```
Comment (achadwick) [inline README.md]: Also it's autotool-dev on Debian testing/unstable right now. No need to special-case Ubuntu.

Comment (shakaran) [inline README.md]: ok, updated ;)

**Comment by odysseywestra** [comment]
@achadwick do you have any more thoughts about this pull request? If not it's probably ready to be merged in. Reason being I've been bit by this too.

**Comment by achadwick** [comment]
Sorry for the delay, I'm still learning the new github patch-review system, it seems. Apparently I have to approve the changes or dismiss the review. I suppose that makes sense.

Anyway, these changes look good, and are needed for both Debian and Ubuntu now, according to `packages.ubuntu.com/PACKAGE` and `packages.debian.org/PACKAGE`. Thanks, @shakaran !

**Comment by shakaran** [comment]
@achadwick it seems that only remain that you click on "merge" pull request since it is approved the review ;)
